src/Controllers/QuestionController.php

```diff
@@ -77,7 +90,7 @@ public function deleteQuestion(RequestInterface $request)
             if ($this->questionService->delete($request)) {
                 $response = new Response(Stream::createFromString(' '), []);
                 $response = $response->withStatus(301);
-                $response = $response->withHeader('Location', 'http://local.quiz.com/question/1');
+                $response = $response->withHeader('Location', 'http://local.quiz.com/question');
 
                 return $response;
```

Comment:
i will use the getRedirectPage method from the abstract controller, but now it's not the goal of the current feature

Reply:
ok